cmd/atlas/internal/sqlparse/sqlparse.go

```diff
@@ -52,5 +52,4 @@ func init() {
 	Register(mysql.DriverName, &myparse.Parser{})
 	Register(postgres.DriverName, &pgparse.Parser{})
 	Register(sqlite.DriverName, &sqliteparse.FileParser{})
-	Register(sqlite.LibsqlDriverName, &sqliteparse.FileParser{})
```

Comment:
Isn't this needed @a8m ?

Reply:
No. For `migrate lint` we recommend users to pass a URL to an in-memory SQLite database. The semantics are the same